### PR TITLE
added cybersource settings to app.json

### LINKED ISSUE #1396: Add elasticsearch and cybersource to default PR app env variables

### PR DESCRIPTION
**Pull request by pdpinch**
#### What are the relevant tickets?

closes #1396 
#### What's this PR do?

Adds cybserource setting names to app.json. If all works as expected, PR builds will pull the values for these settings from the CI server. 
#### How should this be manually tested?

Did the PR build get the appropriate settings?

### CODE REVIEW

**Comment by pdpinch** [comment]
@blarghmatey it works!

but... is there anyway to customize the setting for `CYBERSOURCE_REFERENCE_PREFIX` ? It's not a deal-breaker, but I can imagine some potential confusion if all these PR builds have the same prefix of `ci`

**Comment by noisecapella** [comment]
If they all had the same reference prefix it's possible that an order with the same number on CI might be fulfilled instead of the PR build. Also, we will need to adjust the postback URL for each PR build we want to test

**Comment by noisecapella** [comment]
Environment variables look good, I was able to make it to the cybersource checkout

File: app.json [hunk 1]
```diff
@@ -26,6 +26,24 @@
     "AWS_STORAGE_BUCKET_NAME": {
       "description": "S3 Bucket name."
     },
+	"CYBERSOURCE_ACCESS_KEY": {
+      "description": "Cybersource Access Key"
```
Comment (singingwolfboy) [inline app.json]: Nitpick: Isn't it "CyberSource", with a capital S? (for all of these descriptions)

**Comment by pdpinch** [comment]
I fixed the indenting (oops!) and the spelling. 

I can't think of any good way to deal with the CyberSource test center having the wrong post back URL

**Pull request merged**